.pypirc

```diff
@@ -0,0 +1,6 @@
+[distutils]
+index-servers=
```

Comment:
When release manager tries to do a production release, will this cause a problem?
Will it override their own rc file.